libs/wingsdk/src/shared/misc.ts

```diff
@@ -24,16 +27,14 @@ export function normalPath(path: string) {
  * Just a helpful wrapper around `execFile` that returns a promise.
  */
 export async function runCommand(cmd: string, args: string[]): Promise<any> {
-  const raw = await new Promise((resolve, reject) => {
-    execFile(cmd, args, (error, stdout, stderr) => {
-      if (error) {
-        console.error(stderr);
-        reject(error);
-      }
-      resolve(stdout);
-    });
-  });
-  return raw;
+  try {
+    const { stdout } = await execFilePromise(cmd, args);
+    return stdout;
+  } catch (error) {
+    if (error instanceof Error) {
+      throw error;
+    }
+  }
```

Comment:
Not sure there's value in rethowing since the current callers already are already handling errors

```suggestion
    const { stdout } = await execFilePromise(cmd, args);
    return stdout;
```